src/AutoRest.CSharp/Mgmt/Generation/ResourceContainerWriter.cs

```diff
@@ -119,32 +119,37 @@ private void WriteCheckIfExistsVariants(RestClientMethod getMethod)
 
             _writer.Line();
             _writer.WriteXmlDocumentationSummary($"Tries to get details for this resource from the service.");
-            WriteContainerMethodScope(false, $"bool", "CheckIfExists", passThruParameters, writer =>
+            WriteContainerMethodScope(false, $"Response<bool>", "CheckIfExists", passThruParameters, writer =>
```

Comment:
Should we use `typeof(Response)` here?
```suggestion
            WriteContainerMethodScope(false, $"{typeof(Response)}<bool>", "CheckIfExists", passThruParameters, writer =>
```